src/components/TxData.js

```diff
@@ -93,6 +97,36 @@ class TxData extends React.Component {
     });
   }
 
+  fetchWalletAddressesMap = async () => {
+    const inputs = this.props.transaction.inputs;
+    const outputs = this.props.transaction.outputs;
+
+    const addressWithoutUndecoded = (acc, io) => {
```

Comment:
Why don't you call it `getDecodedAddresses`?

Reply:
Good idea, done in 8886be7fd9049b1078bf4efa3dc8aee7d43a1d5d